oddt/toolkits/rdk.py

```diff
@@ -1253,10 +1277,13 @@ def bonds(self):
     @property
     def partialcharge(self):
         if self.Atom.HasProp('_TriposPartialCharge'):
-            return float(self.Atom.GetProp('_TriposPartialCharge'))
+            return self.Atom.GetDoubleProp('_TriposPartialCharge')
+        if self.Atom.HasProp('_MMFF94Charge'):
```

Comment:
Let's imagine that the user computes mmff94 charges, but then changes his mind decides to use gasteiger instead. He runs `calccharges` method, but then can't use the prefered charges because this method will keep returning mmff94.

Reply:
Also, we have the opposite order in `PDBQTAtomLines`.